docs/en/About.md

```diff
@@ -31,7 +31,7 @@ The ANF now have converted all their EAD and EAC-CPF files metadata to RDF; and
+Of course, French archives, which have encoding practices quite similar to those of the ANF, could be the first institutions interested. However, EAD 2002 and EAC-CPF are now widely used worldwide, by many archive services and by portals or aggregators, as storage and exchange formats. If different practices can be observed there, these institutions share at least the same XML reference models, within which certain elements or attributes are systematically used. We therefore believe that RiC-O Converter can be useful to many institutions other than the ANF. Note that __the conversion of EAD files is done separately from that of EAC-CPF files__. So if you have 'only' EAD files or 'only' EAC-CPF files, you can use it. In addition, __the software, placed under CeCILL-B license (equivalent to MIT license), can be modified to be adapted to a specific project, and also widely redistributed__. It has been precisely documented in English (see the EAD to RiC-O and EAC-CPF to RiC-O [mappings](Mappings.md), and the [unit tests](UnitTests.md)) to facilitate these adaptations. __A good XSLT developer__, knowing the formats of the source metadata, and having taken a little time to understand the essentials of RiC-O, __can easily adapt RiC-O Converter__. Finally, the ANF and Sparna are continuing to develop it. After RiC-O Converter 3, they will most probably work on a v4 of the tool, that will be able to convert born-digital archives in PREMIS. We hope that a larger community will be formed to continue the developments.
```

Comment:
... digital archives using PREMIS ontology

Reply:
or .... descriptions of born digital records conforming to the [SEDA standard](https://francearchives.gouv.fr/fr/article/88482501).